Lodgify.Payments.Stripe.Application/UseCases/GetAccounts/GetAccountsResponse.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace Lodgify.Payments.Stripe.Application.UseCases.GetAccounts;
+
+public record GetAccountsResponse(List<string> StripeAccountIds);
```

Comment:
Soon we will need to extend by adding other properties of account (is onboarding data submitted, can account accept payments). I propose to return list of account objects because of it:
```suggestion
public record GetAccountsResponse(List<AccountResponse> Accounts);
public record AccountResponse(string Id);
```